auk-notebook.ipynb

```diff
@@ -928,6 +928,7 @@
    "source": [
     "# Degree distribution of domains.\n",
     "\n",
+    "# We wrap a few cells below in try/catch statements in-case the Cloud doesn't produce graph files.\n",
```

Comment:
`in-case` to in case

Otherwise comment + explicit exception removal look perfect to me!